std/variant.d

```diff
@@ -1234,6 +1234,22 @@ public:
     assert(a[5] == 7);
 }
 
+@safe version(X86_64) unittest
+{
+    alias Var = VariantN!(24);
+
+    Var a;
+    assert(a.sizeof == 32);
```

Comment:
You could calculate the expected size based on the pointer size:
```suggestion
    assert(a.sizeof == 24 + (void*).sizeof);
```

Reply:
Thanks a lot for this suggestion  (^_^)
Changed the unit test in 6da85c4